piptools/utils.py

```diff
@@ -437,3 +437,10 @@ def get_sys_path_for_python_executable(python_executable: str) -> List[str]:
     assert isinstance(paths, list)
     assert all(isinstance(i, str) for i in paths)
     return [os.path.abspath(path) for path in paths]
+
+
+def remove_value(lst: List[_T], value: _T) -> List[_T]:
+    """
+    Returns new list without a given value.
+    """
+    return [item for item in lst if item != value]
```

Comment:
The function name implies that it will mutate the incoming list which is not true judging from the implementation. Choose a better one.
```suggestion
def omit_list_value(lst: List[_T], value: _T) -> List[_T]:
    """Produce a new list with a given value skipped."""
    return [item for item in lst if item != value]
```

Reply:
Addressed in e361fa5